docs/general/rendering.md

```diff
@@ -243,6 +243,36 @@ This will be rendered as:
 ```
 Note: the `Attributes` adapter (default) does not include a resource root. You also will not be able to create a single top-level root if you are using the :json_api adapter.
 
+#### namespace
+
+The namespace for serializer lookup is based on the controller.
+
+To configure the implicit namespace, in your controller, create a before filter
+
+```ruby
+before_filter :use_my_namespace
+def use_my_namespace
+  self.namespace_for_serializer = Api::V2
+end
```

Comment:
might as well just `before_action {  self.namespace_for_serializer = Api::V2 }`  (before_filter is such old news :( :))